src/include/duckdb/transaction/duck_transaction_manager.hpp

```diff
@@ -41,6 +41,14 @@ class DuckTransactionManager : public TransactionManager {
 		return lowest_active_start;
 	}
 
+	transaction_t CurrentStartTimestamp() {
```

Comment:
Ideally we don't expose these?
I think we use this in one place where we manufacture a transaction because the function call needs it